web/src/components/layout/Layout.tsx

```diff
@@ -1,13 +1,23 @@
 import styles from "./Layout.module.css";
 import Nav from "@/components/nav/Nav";
+import dynamic from "next/dynamic";
+
+const Header = dynamic(() => import("uikit-react-public").then((mod) => mod.Header), {
+  ssr: false,
+});
+const Footer = dynamic(() => import("uikit-react-public").then((mod) => mod.Footer), {
+  ssr: false,
+});
 
 export default function Layout({ children }: { children: React.ReactNode }) {
   return (
     <>
+      <Header title="UCL ARC Services Portal" />
```

Comment:
for me this could probably do with a background slightly taller than it, but that might not be the style 😅

Reply:
you mean cause the bottom of UCL looks like it's cut off?

Reply:
yeah – so it looks like the [front-page](https://www.ucl.ac.uk/) header

Reply:
yeah, they had an AppHeader component but have deprecated (see pic), also our header isn't on the storybook yet so I guess it's still WIP, we can add something in for now but I hope they'll update it later on 
<img width="301" alt="image" src="https://github.com/user-attachments/assets/01405fb7-ede0-4269-8530-4f4e45ade60c" />

Reply:
<img width="832" alt="image" src="https://github.com/user-attachments/assets/c8ff4b44-db41-4d7a-975d-b1778c123ecd" />
?